packages/compiler-dom/__tests__/transforms/vCloak.spec.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  parse,
+  transform,
+  CompilerOptions,
+  ElementNode,
+  NodeTypes
+} from '@vue/compiler-core'
+import { transformCloak } from '../../src/transforms/vCloak'
+
+function transformWithCloak(template: string, options: CompilerOptions = {}) {
+  const ast = parse(template)
+  transform(ast, {
+    directiveTransforms: {
```

Comment:
`directiveTransforms` are called by `transformElement`, so we need to include `transformElement` in `nodeTransforms` here.